docs/best-practices.md

```diff
@@ -0,0 +1,57 @@
+# Implementation Best Practices
+
+This page serves as best practice guidance for CDS Hooks implementers. The best practices outlined here are not mandatory implementation rules; rather, they are suggested guidance. This is a living, in progress document and the best practices outlined here should not be considered absolute or complete.
+
+## Security
+
+Implementing the CDS Hooks security model requires perhaps the most thought and consideration for implementers. Security topics are never binary, are often complex, and robust implementations should factor in concepts such as risk and usability. Implementers should approach their security related development with thoughtful care.
```

Comment:
>Implementing the CDS Hooks security model requires perhaps the most thought and consideration for implementers.

What does this sentence mean? What is it trying to communicate? It should be removed or reworded to be more meaningful.